pkg/archives/archives.go

```diff
@@ -179,3 +179,39 @@ func IsTGZ(b []byte) bool {
 	_, err = tarReader.Next()
 	return err == nil
 }
+
+func CreateFilteredAirgapBundle(airgapBundle string, filesToInclude []string) (io.Reader, error) {
+	buf := bytes.NewBuffer(nil)
```

Comment:
can this write to a file instead of memory? binaries can be large if we add more in the future.